### PR TITLE
Update links and docs

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Library with UI effects
 
-## Basic usage
-        
+## Elements
```
Comment (vergilius) [inline README.md]: its `ui-components`, so maybe just `components` to unify naming?

**Comment by vergilius** [comment]
👍

File: README.md (hunk 1 above)
Comment (matzimowski) [inline README.md]: +1

Comment (aju) [inline README.md]: noooooo It would be components everywhere :(

**Comment by matzimowski** [comment]
👴 would like it as much as he like 👵

**Comment by matzimowski** [comment]
👍

**Comment by kdzwinel** [comment]
👍

**Pull request merged**